docs/concepts/apiserver.md

```diff
@@ -28,12 +28,12 @@ Please see [this](../../example/30-cloudprofile.yaml) example manifest and consu
 Gardener does not care about how a seed cluster got created - the only requirement is that it is of at least Kubernetes v1.20 and passes the Kubernetes conformance tests.
 The Gardener operator has to either deploy the Gardenlet into the cluster they want to use as seed (recommended, then the Gardenlet will create the `Seed` object itself after bootstrapping), or they provide the kubeconfig to the cluster inside a secret (that is referenced by the `Seed` resource) and create the `Seed` resource themselves.
 
-Please see [this](../../example/45-secret-seed-backup.yaml), [this](../../example/50-seed.yaml)(, and optionally [this](../../example/40-secret-seed.yaml)) example manifests.
+Please see [this](../../example/45-secret-seed-backup.yaml), [this](../../example/50-seed.yaml), and optionally [this](../../example/40-secret-seed.yaml) example manifests.
 
 ## Shoot`Quota`s
 
-In order to allow end-users not having their own dedicated infrastructure account to try out Gardener the operator can register an account owned by them that they allow to be used for trial clusters.
-Trial clusters can be put under quota such that they don't consume too many resources (resulting in costs), and so that one user cannot consume all resources on their own.
+In order to allow end-users not having their own dedicated infrastructure account to try out Gardener, the operator can register an account owned by them that they allow to be used for trial clusters.
+Trial clusters can be put under a quota such that they don't consume too many resources (resulting in costs), and so that one user cannot consume all resources on their own.
```

Comment:
```suggestion
To allow end-users not having their dedicated infrastructure account to try out Gardener, the operator can register an account owned by them that they allow to be used for trial clusters.
Trial clusters can be put under quota so that they don't consume too many resources (resulting in costs) and that one user cannot consume all resources on their own.
```